src/App.js

```diff
@@ -3,8 +3,18 @@ import Question from './Question.js';
 import questionData from './multipleQuestions.json';
 import logo from './logo.svg';
 import './App.css';
+import Counter from './Counter.js';
 
 class App extends Component {
+  getInitialState() {
+    return { count: 0 };
+  }
+  handleAnswerSelected(event) {
+    this.setUserAnswer(event.currentTarget.value);
+    if (this.answer.correct) {
+      this.setState = ({count + 1 })
```

Comment:
Again, your instincts are really great here. The handleAnswerSelected function is in the right place (App). This method will need to update this.state.currentQuestion and this.state.correctAnswers. You also did well to pass this into the Question component, but I would pass it in as a handleAnswerSelected prop not count. I think you're having a bit of problems with "this", but the real hint here is that handleAnswerSelected that you pass into the Question component should completely replace the handleClick that is currently in Question.